index.html

```diff
@@ -0,0 +1,190 @@
+                            <p class="price_pakiet pakiet_basic">$0</p>
+                        </div>
+                        <ul class="pricing_list">
+                            <li class="pricing_element pricing_yes">100 MB HDD</li>
+                            <li class="pricing_element pricing_yes">1 Subdomain</li>
+                            <li class="pricing_element pricing_yes">2 E-mails</li>
+                            <li class="pricing_element pricing_not">Two years license</li>
+                            <li class="pricing_element pricing_not">Full support</li>
+                        </ul>
+                        <a href="#" class="btn btn_basic">Begin</a>
+                    </div>
+                <div class="pricing_box pricing_professional">
+                    <h3 class="headline_pricing_box">Professional</h3>   
+                    <p class="price_pakiet pakiet_professional">$25</p>  
+                    <p class="limited_info">Limited offer</p>
+                    <ul class="pricing_list">
+                        <li class="pricing_element pricing_yes">500 MB HDD</li>
+                        <li class="pricing_element pricing_yes">2 Subdomains</li>
+                        <li class="pricing_element pricing_yes">5 E-mails</li>
+                        <li class="pricing_element pricing_yes">One year license</li>
+                        <li class="pricing_element pricing_not">Full support</li>
+                    </ul>
+                    <a href="#" class="btn btn_professional">Begin</a>
+                </div>
+                <div class="pricing_box pricing_premium">
+                    <h3 class="headline_pricing_box">Premium</h3> 
+                    <p class="price_pakiet pakiet_basic">$60</p>
+                    <p class="limited_info">Limited offer</p>
+                    <ul class="pricing_list">
+                        <li class="pricing_element pricing_yes">2 GB HDD</li>
+                        <li class="pricing_element pricing_yes">5 Subdomains</li>
+                        <li class="pricing_element pricing_yes">20 E-mails</li>
+                        <li class="pricing_element pricing_yes">Two years license</li>
+                        <li class="pricing_element pricing_yes">Full support</li>
+                    </ul>
+                    <a href="#" class="btn btn_premium">Begin</a>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="section_contact">
+        <div class="contant_container">
+            <div class="contact_info">
+                <h2 class="headline_questions">Any questions?</h2>
+                <p class="questions_text">Leave your email address or call us!</p>
+                    <div class="contant_box">
+                        <span class="contact_detalis email_icon">info@bestshop.xyz</span>
+                        <span class="contact_detalis phone_icon">123 456 789</span>
```

Comment:
mozna bylo dać to w anchor tag i tel prefix.
cost akiego https://stackoverflow.com/questions/17219688/href-tel-and-mobile-numbers